Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -412,6 +421,10 @@ void IMixedRealityTouchHandler.OnTouchCompleted(HandTrackingInputEventData event
         {
             if (touchPoints.ContainsKey(eventData.Controller))
             {
+                // Giving a last chance to check if pressed ocurred before another Near interaction becomes the closest to pointer
```

Comment:
I would change this comment to something more generic as it doesn't only apply to when you get close to another near interaction touchable:
How about:
// Giving last chance to check if a press has occurred before loosing focus

Reply:
Changed